### DISCUSSION TITLE
Mute the microphone

### DISCUSSION THREAD

**Discussion by PythonNube** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#85] [Q&A] [answered]
This makes the S3-Box soo much better!  One issue I have is my wake work sometimes is triggered in meetings I am in by my speakers.  I would like to mute the microphone.  In the docs you say don't use the mute button on top but use the one on the screen.   The one on the screen seems to mute the speaker and not the microphone.  Am I missing something?

**Comment by BigBobbas** [comment, marked as answer]
Hi There ,
there is a newer version of the firmware which fixes an issue with the mute and makes it clearer as to which icon is to mute speakers and which is to mute mic.
the best config to use at the moment is this one https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/blob/dev/s3b_dev.yaml
if you need any further help just let me know or join us on discord https://discord.gg/U3SRYCG6Kp

cheers and thanks for your support in trying out the project 🙏 

Rob

**Comment by PythonNube** [comment]
Just updated my config and it works great.  Once I get a chance to play with it some, I will see if I can dive in and help.  Thanks for all the work on this one!
